contrib/chart/backstage/templates/lighthouse-deployment.yaml

```diff
@@ -51,6 +51,7 @@ spec:
                   name: {{ include "lighthouse.postgresql.passwordSecret" . }}
                   key: postgresql-password
 
+      {{- if .Values.lighthouse.postgresCertMountEnabled }}
```

Comment:
nitpick: might be good for avoiding potential mistakes in future additions to this file to add two if-elses like in the backend part (one for lines 55-56 and another for  lines 59-61). Just a suggestion.

Reply:
added 2 "if" but before "volumeMounts" and "volumes" as there is only 1 element in the list, in other way we need add "[]" to "else" which also looks not good

Reply:
Right :+1:  looks good